gtsfm/utils/ellipsoid.py

```diff
@@ -0,0 +1,103 @@
+"""Algorithms to fit Ellisoid geometry to 3D points and extract the rotation matrix necessary to align point cloud 
+with x, y, and z axes. Used in React Three Fiber Visualization Tool.
+
+Authors: Adi Singh
+"""
+
+import numpy as np
+from numpy.linalg import inv, norm, svd
+
+
+def center_point_cloud(point_cloud: np.ndarray) -> np.ndarray:
+    """Centers a point cloud using mean values of x, y, and z.
+
+    Args:
+        pointCloud: array representing the point cloud, of shape N x 3.
+
+    Returns:
+        points_centered: centered point cloud of shape Nx3.
+        means: the means of the x,y,z coordinates, array of length 3.
```

Comment:
maybe prefer `mean: array of shape (3.) representing the mean x,y,z, coordinates of point cloud`